app/src/test/java/com/android/periodpals/model/authentication/AuthenticationModelSupabaseTest.kt

```diff
@@ -178,4 +184,34 @@ class AuthenticationModelSupabaseTest {
         onFailure = { assert(true) },
     )
   }
+
+  @Test
+  fun `login with google success`() = runBlocking {
+    `when`(auth.signInWith(IDToken)).thenReturn(Unit)
+
+    var successCalled = false
+    authModel.loginGoogle(
+        idToken,
+        rawNonce,
+        { successCalled = true },
+        { fail("Should not call onFailure") },
```

Comment:
To be consistent with other models' tests, this should be as follow :
```suggestion
        { fail("Should not call onFailure") },
```
Same for `onSuccess`. It would be nice if you could change it in the other tests of this class.

Else, LGTM.